src/phoenix/server/api/input_types/PromptVersionTagInput.py

```diff
@@ -0,0 +1,23 @@
+from typing import Optional
+
+import strawberry
+from strawberry import UNSET
+from strawberry.relay import GlobalID
+
+
+@strawberry.input
+class CreatePromptVersionTagInput:
+    """Fields required to create a PromptVersionTag."""
+
+    name: str
+    description: Optional[str] = None
+    prompt_id: GlobalID
+
+
+@strawberry.input
+class PatchPromptVersionTagInput:
+    """Fields that can be updated on a PromptVersionTag."""
+
+    version_tag_id: GlobalID
+    name: Optional[str] = UNSET
+    description: Optional[str] = UNSET
```

Comment:
consolidate into 

```graphql

@stawberry.input
class SetPromptVersionTagInput:

   prompt_version_id: GlobalID
   name: str
   description: Optional[str] 
   
   ```
   
   Where description, if provided re-writes the description